README.rst

```diff
@@ -111,6 +111,33 @@ We welcome pull requests. Please see our `developer documentation
 page <https://glm-tools.github.io/pyglmnet/contributing.html>`__ for more
 details.
 
+Citation
+~~~~~~~~
+
+If you use ``pyglmnet`` package in your publication, please cite use from
+our `JOSS publication <https://doi.org/10.21105/joss.01959>`__ using the following BibTex
+
+.. code::
+
+   @article{Jas2020,
+   doi = {10.21105/joss.01959},
+   url = {https://doi.org/10.21105/joss.01959},
+   year = {2020},
+   publisher = {The Open Journal},
+   volume = {5},
+   number = {47},
+   pages = {1959},
+   author = {Mainak Jas and Titipat Achakulvisut and Aid Idrizović
+             and Daniel Acuna and Matthew Antalek and Vinicius Marques
+             and Tommy Odland and Ravi Garg and Mayank Agrawal
+             and Yu Umegaki and Peter Foley and Hugo Fernandes
+             and Drew Harris and Beibin Li and Olivier Pieters
+             and Scott Otterson and Giovanni De Toni and Chris Rodgers
+             and Eva Dyer and Matti Hamalainen and Konrad Kording and Pavan Ramkumar},
+   title = {Pyglmnet: Python implementation of elastic-net regularized generalized linear models},
```

Comment:
You will lose the capitalization in this. need extra curly braces to take care of it